author.tf

```diff
@@ -315,6 +315,7 @@ module "author-api" {
   ecs_alb_security_group     = ["${module.author-eq-ecs.ecs_alb_security_group}"]
   launch_type                = "FARGATE"
   alb_listener_path_patterns = ["/graphql*", "/launch*", "/status", "/export*", "/import*", "/signIn"]
+  aws_alb_use_host_header    = false
```

Comment:
Is this something the aws module provides or does it require another change?

Reply:
This is a variable exposed by `eq-ecs-deploy` it decides whether the listener rule is created with the host attribute or not.

Reply:
instead of passing a boolean, can we pass a wildcard eg `alb_host_header = "*"` to just override a default?